package.json

```diff
@@ -3,6 +3,9 @@
   "version": "3.0.1",
   "description": "One place to hold all the logic to download and run ember tests in chromium",
   "main": "index.js",
+  "bin": {
+    "install-chromium": "download-chrome.js"
```

Comment:
I went back and forth on if we should use a different name for the linked script.  On one hand, renaming would make for an easy migration path for existing users.  On the other, it would still work if they keep their local npm script and `install-chromium` is probably already the best name for this script.

Open to thoughts though.